docs/application/native/guides/multimedia/image-edit.md

```diff
@@ -471,7 +466,47 @@ To encode a raw image:
    ```
 
 <a name="animation"></a>
-## Encoding an Animated GIF
+## Encoding an Animated GIF or WEBP
+
+To encode an animated GIF or WEBP image, follow these steps:
+
+1. Create an encoding handle using `image_util_anim_encode_create()`:
+
+   ```
+   image_util_anim_encode_h anim_encode_h = NULL;
+   ret = image_util_anim_encode_create(IMAGE_UTIL_ANIM_WEBP, &anim_encode_h);
+   ```
+
+2. Set the loop count, background color or lossless `image_util_anim_encode_set_loop_count()`, `image_util_anim_encode_set_background_color()` or `image_util_anim_encode_set_lossless()`:
+
+   ```
+   ret = image_util_anim_encode_set_loop_count(anim_encode_h, 10);
+   ```
+
+   > [!NOTE]
+   > The background color and lossless compression is supported for the WEBP animation.
+   > The default of the loop count is '0' (infinite), and the default of lossless compression is 'false' (lossy).
+   > The default background color is 'no use'.
```

Comment:
```suggestion
   > [!NOTE]
   > The background color and the lossless compression is supported for the WEBP animation.
   > The default of the loop count is `0`(infinite), and the default of lossless compression is `false` (lossy).
   > The default background color is `no use`.

Reply:
OK, 'the' article has been added, others were modified.